lms/static/js/spec/student_account/register_spec.js

```diff
@@ -502,6 +513,27 @@
                     expect(view.$submitButton).toHaveAttr('disabled');
                 });
 
+                it('shows optional not toggled fields', function() {
+                    var formFields = FORM_DESCRIPTION.fields
+                    formFields.push({
+                        placeholder: '',
+                        name: 'not_toggled_custom_optional_field',
+                        label: 'Not toggled custom optional field.',
+                        defaultValue: '',
+                        type: 'checkbox',
+                        required: false,
+                        toggled: false,
+                        instructions: 'Check this field if you would like to.',
+                        restrictions: {}
+                    })
+
+                    createRegisterView(this, formFields);
+                    var elementClasses = view.$('.not-toggled-optional-fields').attr('class');
+                    // Expect the toggled optional fields container does not have other
+                    // classes assigned, like .hidden
+                    expect(elementClasses).toEqual('not-toggled-optional-fields');
```

Comment:
If I understand correctly, we're only checking that the `.not-toggled-optional-fields` div exists on the page. However, we should also verify that it has visible children.

Reply:
You are probably right. I'll add that.